bin/index.js

```diff
@@ -13,6 +13,8 @@ const shell = require('shelljs');
 const proxy = require('../lib/proxy');
 const browserSync = require('../lib/browserSync');
 const findPort = require('../lib/findPort');
+const dotenv = require('dotenv');
+const env = dotenv.config().parsed;
```

Comment:
WDYT about extending proxyConfig by siteName?

Reply:
I left it out as to me it isn't really a configuration for the proxy, rather just an environment config